requirements.txt

```diff
@@ -1,4 +1,5 @@
 fabric>=1.8.2
+colorlog
```

Comment:
Not sure where this is used (at least this library is not used in the code). Please suggest.